Dockerfile.mysql

```diff
@@ -2,10 +2,7 @@
 FROM mysql:latest
 
 # Environment variables for MySQL configuration
-ENV MYSQL_ROOT_PASSWORD=1234
-ENV MYSQL_DATABASE=app_db
-ENV MYSQL_USER=app_user
-ENV MYSQL_PASSWORD=1234
+
```

Comment:
It's crucial to define environment variables for MySQL configuration, such as `MYSQL_ROOT_PASSWORD`, `MYSQL_DATABASE`, `MYSQL_USER`, and `MYSQL_PASSWORD`. Ensure these are set securely, preferably using Docker secrets or environment variables passed at runtime to avoid hardcoding sensitive information.